packages/forms/src/UIForm/fields/MultiSelectTag/MultiSelectTag.component.test.js

```diff
@@ -66,7 +66,7 @@ describe('MultiSelectTag field', () => {
 
 		// then
 		expect(screen.getByRole('listbox')).toBeInTheDocument();
-		expect(screen.getByRole('option', { name: 'totomobile' })).toBeInTheDocument();
+		expect(screen.getByTitle('totomobile')).toBeInTheDocument();
```

Comment:
this is a feature from https://github.com/eps1lon/dom-accessibility-api/pull/893